Blocks/Persistence/file.h

```diff
@@ -43,6 +43,8 @@ SOFTWARE.
 #include "../../Bricks/time/chrono.h"
 #include "../../Bricks/sync/scope_owned.h"
 
+#include "../../Bricks/util/atomic_that_works.h"  // TODO(dkorolev): Remove once GCC 5 is mainstream.
```

Comment:
I did some research and the main problem is not the GCC version. The thing is that `std::pair` is not "trivially copyable" (which is pretty weird platform-dependent concept). The best way I found to make it work both with Clang/OSX and GCC 4.8.2/Ubuntu is this one:
```
struct pod_idxts_t {
  uint64_t index;
  std::chrono::microseconds us;
  pod_idxts_t() = default;
  pod_idxts_t(uint64_t index, std::chrono::microseconds us) : index(index), us(us) {}
};
struct pod_idxts_pair_t {
  pod_idxts_t first;
  pod_idxts_t second;
};
std::atomic<pod_idxts_t> a1;
std::atomic<pod_idxts_pair_t> a2;
```
**NOTE:** GCC 4.8.2 requires `-mcx16` flag.